Generators/src/main/java/io/deephaven/plot/util/GeneratePyV2FigureAPI.java

```diff
@@ -24,7 +24,7 @@
     private static final String INDENT = "    ";
     private static final String JCLASS = "io.deephaven.plot.Figure";
     private static final String PYPREAMBLE = "/Generators/src/main/java/io/deephaven/pythonPreambles/plotV2.py";
-    private static final String PYMODUlE = "/pyintegration/deephaven2/plot/figure.py";
+    private static final String PYMODUlE = "/pyintegration/deephaven/plot/figure.py";
```

Comment:
Possibly rename this class too? and also the various gradle tasks that are specific to deephaven vs deephaven2?

Reply:
Description covers this, i'll ignore for now, please make a follow up

Reply:
This can be done after we officially kill deephavenv1, most of the generator stuff can be removed